packages/spark-core/base/_inputs.scss

```diff
@@ -0,0 +1,105 @@
+// ==================================================================
+// Input Component
+// ==================================================================
+label,
+.sprk-b-Label {
+  @include TextCrop(
+    $line-height: $line-height-display-seven,
+    $top-crop: 8,
+    $bottom-crop: 8,
+    $crop-font-size: (strip-unit($font-size-display-seven) * 16),
+    $crop-line-height: $line-height-display-seven);
+
+  font-size: $font-size-display-seven;
+  color: $color-display-seven;
+  font-weight: $font-weight-display-seven;
+  line-height: $line-height-display-seven;
+}
+
+.sprk-b-Label--inline {
+  display: inline-block;
+}
+
+.sprk-b-SubLabel {
+  font-size: $font-size-body-two;
+  font-weight: $font-weight-body-two;
```

Comment:
scales has .FormLabel--additional which, sets the same font-weight you're applying of "normal", that font-weight declaration can be removed